pages/user guides/financial_impact.md

```diff
@@ -0,0 +1,88 @@
+
+{% include image.html file="financial-impact-6.png" %}
+
+After you've created your custom field, you can navigate back to the app record and populate it with data. Your financial impact visuals will be updated appropriately to reflect this new current or cloud cost.
+
+{% include image.html file="financial-impact-7.png" %}
+
+{% include note.html content="To be included in an app's current or cloud costs, the field must have category 'Financial' or 'Transition Planning' (for current and cloud costs respectively), and must have the field type 'Currency'. In most cases, you'll also want to set the field aggregation to 'Sum'." %}
+
+## Application Cost Breakdown
+
+Building on what you've learned from creating current and cloud cost fields above, you can leverage our financial classification system for further granularity over your application's current and cloud costs.
+
+When creating a custom field where the category is 'Financial' or 'Transition Planning' and the field type is 'Currency', you'll notice that there is an additional attribute available - 'Financial Classification'. This allows you to distinguish between different cost types, such as license or maintenance costs. 
+
+{% include image.html file="financial-impact-8.png" %}
+
+By ensuring these are set correctly, you will be able to generate a cost breakdown for each of your apps, which gives you in-depth insight in the cost of your app when comparing current, cloud and transformative costs. This table can be found in the application drill-down (or on the app itself) under 'Cost Breakdown'.
+
+{% include image.html file="financial-impact-9.png" %}
+
+This dovetails with the custom field system outlined above. If you need to add a cost which is not covered by our default fields, then your custom costs will appear on the app record, in our charts and summaries, and here in the cost breakdown table.
+
+### Detailed Cost Breakdown
+
+The app record's financial impact view also has a 'Detailed Cost Breakdown' table, which lays out all cost sources for your app, including fields and cloud instance targets. This can be found by navigating to the app record, then the 'Financial Impact' tab, under the 'Cost Breakdown' section. You will be able to tab between the cost breakdown as described above, and also the detailed cost breakdown, which provides more granularity and insight into an app's cost sources.
+
+## ROI Comparison
+
+Part of building your cloud migration business case is comparing and contrasting different ** 6R ** approaches to cloud migration. Tidal Accelerator's financial impact view includes a visual comparing the ROI of different possible migration strategies (also known as transition types). This can be found in the app drill-down under 'Recommendation Details'. To populate this, on the app record under 'Transition Planning' you will find the following fields; 'Repurchase Cost', 'Replatform Cost' and 'Transformative Cost'. 
```

Comment:
I think it's good form to show the unabbreviated form before you abbreviate it. 
>... return on investment (ROI)
and then switch to ROI for the rest of the document.